widget/view.controller.js

```diff
@@ -6,85 +6,404 @@
+        $scope.triggerSchedule = triggerSchedule;
+        $scope.backStartPage = backStartPage;
+        $scope.configHuntTool = configHuntTool;
+        $scope.onlyNumbers = '^0*[1-9][0-9]*\d*$';
+        $scope.backSelectHuntTools = backSelectHuntTools;
+        $scope.threatHuntSchedule = threatHuntSchedule;
+        $scope.backThreatHuntConfig = backThreatHuntConfig;
+        $scope.backThreatHuntSchedule = backThreatHuntSchedule;
+        $scope.moveToFinish = moveToFinish;
         $scope.close = close;
-        $scope.moveNext = moveNext;
-        $scope.moveTktNext = moveTktNext;
-        $scope.movePrevious = movePrevious;
-        $scope.moveTktPrevious = moveTktPrevious;
-        $scope.moveEnvironmentNext = moveEnvironmentNext;
+        $scope.saveConnector = saveConnector;
+        $scope.loadActiveTab = loadActiveTab;
+        $scope.toggle = [];
+        $scope.toggleRemediation = true;
+        $scope.toggleConnectorConfig = [];
+        $scope.toggleRemediationConfig = false;
+        $scope.toggleAdvancedSettings = toggleAdvancedSettings;
+        $scope.toggleConnectorConfigSettings = toggleConnectorConfigSettings;
+        $scope.backNotification = backNotification;
+        $scope.nextNotification = nextNotification;
+        $scope.isLightTheme = $rootScope.theme.id === 'light';
+        $scope.widgetBasePath = widgetBasePath;
+        $scope.startInfoGraphics = $scope.isLightTheme ? widgetBasePath + 'images/start-light.svg' : widgetBasePath + 'images/start-dark.svg';
+        $scope.selectThreadHuntTool = $scope.isLightTheme ? widgetBasePath + 'images/select-threat-hunt-tool-light.svg' : widgetBasePath + 'images/select-threat-hunt-tool-dark.svg';
+        $scope.threatHuntToolConfig = $scope.isLightTheme ? widgetBasePath + 'images/threat-hunt-config-light.svg' : widgetBasePath + 'images/threat-hunt-config-dark.svg';
+        $scope.outbreakSettings = $scope.isLightTheme ? widgetBasePath + 'images/remediation-light.svg' : widgetBasePath + 'images/remediation-dark.svg';
+        $scope.threatTuntSchedule = $scope.isLightTheme ? widgetBasePath + 'images/threat-hunt-schedule-light.svg' : widgetBasePath + 'images/threat-hunt-schedule-dark.svg';
+        $scope.finishInfoGraphics = widgetBasePath + 'images/finish.png';
+        $scope.widgetCSS = widgetBasePath + 'widgetAsset/wizard-style.css';
+        $controller('BaseConnectorCtrl', {
+            $scope: $scope
+        });
+        $scope.autoInstall = {
+            enabled: true
+        };
         $scope.selectedEnv = {
             huntTools: [],
-            ticketTools: []
+            installOutbreakType: [],
+            threatHuntToolsParams: {
+                fazParams: null,
+                fsmParams: null,
+                qRadarParams: null,
+                splunkParams: null
+            },
+            timeFrameDays: null,
+            emailAddress: null
         };
+        $scope.severityValues = ['Critical', 'High', 'Medium'];
```

Comment:
if its a OA severity then var name could be outbreakAlertSeverityList

Reply:
done